app_dart/lib/src/request_handlers/check_flaky_builders.dart

```diff
@@ -81,6 +81,9 @@ class CheckFlakyBuilders extends ApiRequestHandler<Body> {
           await bigquery.listRecentBuildRecordsForBuilder(kBigQueryProjectId, builder: info.name, limit: kRecordNumber);
       if (_shouldDeflake(builderRecords)) {
         await _deflakyPullRequest(gitHub, slug, info: info, ciContent: ciContent, testOwnership: testOwnership);
+        // Manually add a 1s delay between consecutive GitHub requests to deal with secondary rate limit error.
+        // https://docs.github.com/en/rest/guides/best-practices-for-integrators#dealing-with-secondary-rate-limits
+        await Future.delayed(const Duration(seconds: 1));
```

Comment:
This should be injectable so tests can use `Duration.zero`

Reply:
Defined in config so that tests could use default zero from fake.